bittensor/utils/__init__.py

```diff
@@ -220,7 +220,7 @@ def run(self):
                 with self.check_block:
                     block_number = self.curr_block_num.value
                     block_bytes = bytes(self.curr_block)
-                    block_difficulty = int(self.curr_diff[0] >> 32 | self.curr_diff[1])
+                    block_difficulty = int(self.curr_diff[0] << 32 | self.curr_diff[1])
```

Comment:
is this covered with some unit tests??
Seems like it's the place to catch these details

Reply:
From my point of view, when a piece of code is that critical, it's worthy to extract into a new function a cover it with unit tests. In this way, integration tests could avoid knowing about its domain.

Reply:
the integration test covers it. Perhaps I should move the test to the unit test folder as it's more of a unit test.